src/SIM/SIMoutput.C

```diff
@@ -333,7 +333,7 @@ bool SIMoutput::writeGlvBC (int& nBlock, int iStep) const
 
     geomID++;
     size_t nbc = myModel[i]->getNoFields(1);
-    Matrix bc(nbc,myModel[i]->getNoNodes(-1));
+    Matrix bc(nbc,myModel[i]->getNoNodes(1));
```

Comment:
This one actually not so great. It causes one (or at least one) of the FiniteDeformation tests with contact to crash, because the bc array now becomes too small. Was not caught because these tests have not been in the loop after the repo-split.